src/media/css/footer.styl

```diff
@@ -45,6 +45,11 @@ $support-nav-height = 55px;
             display: block;
         }
     }
+
+    .page-footer-text {
+        color:$action-neutral;
```

Comment:
Normally we'd leave a space after the colon `color: $action-neutral;` but it's just a minor nitpick.